src/1.2/index.ts

```diff
@@ -0,0 +1,82 @@
+import { constants, createReadStream, createWriteStream, WriteStream } from 'fs'
+import { access } from 'fs/promises'
+import path from 'path'
+import csv from 'csvtojson'
+import { URL } from 'url'
+
+const __dirname = new URL('./', import.meta.url).pathname
```

Comment:
I see that usage of global variable __dirname in Node returns the same result (at least in Windows). What was the reason to introduce such way of reading a directory name?

Reply:
As I understood, when I switched `type` to `module` in package.json, I have no variables `__dirname` and `__filename` available anymore. And I've made my homework on macOS.

Reply:
I see now. Not sure that type module has some impact since you are using Typescript. I assume that it's more related to direct ES6 module usage in Node 14+ version.

Reply:
Yeah, but imports didn't work without that. What I've done wrong?) Node 16